src/Pretzel.Logic/Templating/Context/SiteContextGenerator.cs

```diff
@@ -186,12 +186,12 @@ private bool ContainsYamlFrontMatter(string file)
             return postFirstLine != null && postFirstLine.StartsWith("---");
         }
 
-        private bool IsExcludedPath(string relativePath)
+        public bool IsExcludedPath(string relativePath)
```

Comment:
If it's public you have to make sure that there is the corresponding unit tests.